infra/templates/azure-simple/README.md

```diff
@@ -36,33 +36,51 @@ Application developer team that want to provision a secure app service plan in a
 
 ## Example Usage
 
-1. Create or switch to the target workspace.
+1. Execute the followign commands to set up your local environment variables:
 
-```hcl
+```bash
+# these commands setup all the environment variables needed to run this template
+DOT_ENV=<path to your .env file>
+export $(cat $DOT_ENV | xargs)
+```
+
+2. Execute the following commands to set up your terraform environment
+```bash
+# Variables prefixed with TF_VAR_ will be picked up by terraform
+export TF_VAR_remote_state_account=<tf-remote-state-storage-account-name>
```

Comment:
Do we need to define `TF_VAR_remote_state_account` as it's included in `.env.template`?

Reply:
nope, good call. I'll remove these 2 vars as they aren't needed.